testsuite/features/support/constants.rb

```diff
@@ -967,6 +982,11 @@
         sle-manager-tools-for-micro5-pool-x86_64-5.5
         sle-manager-tools-for-micro5-updates-x86_64-5.5
       ],
+    'sl-micro-6.0' => # TODO: Enable when it becomes available sl-manager-tools-for-micro6-updates-x86_64-6.0
+      %w[
+        sl-manager-tools-for-micro6-pool-x86_64-6.0
```

Comment:
I guess we need to rename this following these channel names?

![image](https://github.com/uyuni-project/uyuni/assets/2827771/28f881f2-06ae-4387-a551-097324402c4b)

Reply:
checked and corrected, will test as soon as the port is also adjusted